polaris-react/src/components/Box/Box.scss

```
@@ -0,0 +1,172 @@
+.root {
+  --pc-box-margin: initial;
+  --pc-box-margin-bottom: initial;
+  --pc-box-margin-left: initial;
+  --pc-box-margin-right: initial;
+  --pc-box-margin-top: initial;
+  --pc-box-padding: initial;
+  --pc-box-padding-bottom: initial;
+  --pc-box-padding-left: initial;
+  --pc-box-padding-right: initial;
+  --pc-box-padding-top: initial;
+
+  display: block;
+  margin: var(--pc-box-margin);
+  margin-bottom: var(--pc-box-margin-bottom);
+  margin-left: var(--pc-box-margin-left);
+  margin-right: var(--pc-box-margin-right);
+  margin-top: var(--pc-box-margin-top);
+  padding: var(--pc-box-padding);
+  padding-bottom: var(--pc-box-padding-bottom);
+  padding-left: var(--pc-box-padding-left);
+  padding-right: var(--pc-box-padding-right);
+  padding-top: var(--pc-box-padding-top);
```

We could remove the above lines of code by saying if the custom property isn't defined then it falls back to `initial`.

```suggestion
  margin: var(--pc-box-margin, initial);
```